ntoskrnl/mm/ARM3/expool.c

```diff
@@ -489,7 +489,7 @@ ExpComputePartialHashForAddress(IN PVOID BaseAddress)
 #endif
 
 VOID
-MiDumpPoolConsumers(BOOLEAN CalledFromDbg, ULONG Tag, ULONG Mask, ULONG Flags)
+MiDumpPoolConsumers(BOOLEAN CalledFromDbg, ULONG Tag, ULONG Mask, ULONG_PTR Flags)
```

Comment:
What's the point of this? Flags are usually not dependent on pointer size.

Reply:
Yeah, but `ExpKdbgExtPoolUsed()` is calling it with a `ULONG_PTR` value.
If I remember correctly, I initially submitted a cast there, which was not committed.
A more proper solution would be to create a `KdbpGetHexNumberUlong()`.

Reply:
Please add the cast instead.

Reply:
Done, and I added a few `ASSERT()`.